README.md

```diff
@@ -249,4 +249,4 @@ Polkadot is [GPL 3.0 licensed](LICENSE).
 
 ## Important Notice
 
-https://polkadot.network/testnetdisclaimer
+<https://polkadot.network/testnetdisclaimer>
```

Comment:
is this link still relevant? probably should be removed along with `Important Notice`

Reply:
The idea was to get spellchecking to pass, not so much on a general cleanup, but yes, this should be removed.